### DISCUSSION TITLE
Icon white insted of black in MacOS Sonoma

### DISCUSSION THREAD

**Discussion by JackobStuard** [rustdesk/rustdesk#6642] [General]
### Bug Description

On MacOS Sonoma 14.1.2 toolbar icon is white instead of black, is possible to change that somehow?

### How to Reproduce

On app open in toolbar is possible to see white icon instead of black?

### Operating system(s) on local side and remote side

MacOS

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

IMG https://prnt.sc/wXZRCRCRDomc 3rd icon

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/fc110c4988a4ec347256d8412eebb6d8dd7508e3
